readme.md

```diff
@@ -81,7 +79,9 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+
+geraldo = students.create(name: "Geraldo")
+sandwiches.create(type: 'pbj', student: geraldo)
 
```

Comment:
I don't think the create method exists for an active record relation. You could do something like `Student.create`. Second part looks fine.